app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -429,14 +459,19 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     if ($scope.emsCommonModel.ssh_keypair_auth_status === true) {
       $scope.postValidationModelRegistry("ssh_keypair");
     }
+    if ($scope.emsCommonModel.prometheus_alerts_auth_status === true) {
+      $scope.postValidationModelRegistry("prometheus_alerts");
+    }
   };
 
   $scope.postValidationModelRegistry = function(prefix) {
     if ($scope.postValidationModel === undefined) {
-      $scope.postValidationModel = {default: {},
-                                    amqp: {},
-                                    metrics: {},
-                                    ssh_keypair: {}
+      $scope.postValidationModel = {
+        default: {},
+        amqp: {},
+        metrics: {},
+        ssh_keypair: {},
+        prometheus_alerts: {}
```

Comment:
trailing comma is missing

Reply:
👍 Thanks ! how did this pass code climate ?